react/features/connection-indicator/components/web/ConnectionIndicator.js

```diff
@@ -151,13 +162,19 @@ class ConnectionIndicator extends AbstractConnectionIndicator<Props, AbstractSta
                 className = { rootClassNames }
                 content = { <ConnectionIndicatorContent
                     inheritedStats = { this.state.stats }
-                    participantId = { this.props.participantId } /> }
-                disablePopover = { !this.props.enableStatsDisplay }
-                position = { this.props.statsPopoverPosition }>
+                    participantId = { participantId } /> }
+                disablePopover = { !enableStatsDisplay }
```

Comment:
nit: indentation is a bit off the mark

Reply:
Hmmm, this is 16 spaces just like the `content` and `className` props. The previous line has indentation of 20 because the props belong to another component. Or am I missing something? 🤔

Reply:
nevermind, for some reason i thought props starting from `disablePopover` also belong to `ConnectionIndicatorContent`